packages/context/test/unit/context.ts

```diff
@@ -375,6 +375,23 @@ describe('Context', () => {
     });
   });
 
+  describe('toJSON()', () => {
+    it('converts to plain JSON object', () => {
+      ctx.bind('a').to('1').lock();
+      ctx.bind('b').toDynamicValue(() => 2).inScope(BindingScope.SINGLETON)
+        .tag(['X', 'Y']);
+      ctx.bind('c').to(3).tag('Z');
```

Comment:
`.to()` and `.toDynamicValue()` calls are redundant, because the JSON output does not include any information about the kind of the binding. Please remove them.

Alternatively, we may want to start capturing the kind of the value the binding is producing (a static value via `.to()`, a dynamic value via `.toDynamicValue()`, a class instance via `.toClass()`, a dynamically provided value via `.toProvider()`) , in which case it will make sense to write tests verifying how are these different kinds of values represented in the test output.

Also since we already have tests verifying how `Binding` is converted to JSON, there is no need to verify that part in the `Context` tests again. In this test, it should be enough to create a single binding and then verify the representation of key/binding structure in JSON.